### PR TITLE
VIH-7638 Review and Repair Failing Tests

### PR DESCRIPTION
**Pull request by ibthmcts**
… can be printed when logging

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
VIH-7638

### Change description ###
Convert 'expected states' collection to an array so that the contents can be printed when logging

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by MichaelKeaneathmcts — APPROVED** [review]

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/CommonSteps.cs [hunk 1]
```diff
@@ -25,6 +25,9 @@ public CommonSteps(TestContext testContext, Dictionary<UserDto, UserBrowser> bro
         [When(@"the user clicks the (.*) button")]
         public void WhenTheUserClicksTheButton(string label)
         {
+            if (label.Equals("Check equipment again", System.StringComparison.OrdinalIgnoreCase))
+                _browsers[_c.CurrentUser].Driver.WaitUntilVisible(CommonLocators.ElementContainingText("Please wait to be connected")).Displayed.Should().BeTrue();
```
Comment (MichaelKeaneathmcts) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/CommonSteps.cs:29]: Is there no identifier for the web element?

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/CommonSteps.cs [hunk 2]
```diff
@@ -25,6 +25,9 @@ public CommonSteps(TestContext testContext, Dictionary<UserDto, UserBrowser> bro
...
+
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(CommonLocators.ButtonWithInnerText(label)).Displayed.Should().BeTrue();
```
Comment (MichaelKeaneathmcts) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/CommonSteps.cs:31]: Is there no identifier for the web element?

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/JudgePrivateConsultationSteps.cs [hunk 3]
```diff
@@ -86,7 +86,7 @@ public void WhenTheyEnterPrivateConsultationRoom()
         {
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(JudgeWaitingRoomPage.HearingTitle).Displayed.Should().BeTrue();
             WaitForUserStatusToBe(ParticipantState.Available);
-            _browsers[_c.CurrentUser].ClickToProgress(JudgeWaitingRoomPage.EnterPrivateConsultationButton, PrivateConsultationRoomPage.LeavePrivateConsultationButton, 10);
+            _browsers[_c.CurrentUser].ClickToProgress(JudgeWaitingRoomPage.EnterPrivateConsultationButton, PrivateConsultationRoomPage.LeavePrivateConsultationButton);
```
Comment (MichaelKeaneathmcts) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/JudgePrivateConsultationSteps.cs:89]: Has this been tested in a pipeline?
There should have been a reason for setting the timeout override.

**Review by MichaelKeaneathmcts — CHANGES_REQUESTED** [review]

**Comment by MichaelKeaneathmcts** [comment]
VIH-7638 is the main task for all repairs, did you create a sub-task for this fault?

**Comment by ibthmcts** [comment]
> [VIH-7638](https://tools.hmcts.net/jira/browse/VIH-7638) is the main task for all repairs, did you create a sub-task for this fault?

No. Rather all script repairs are committed to a single branch so a history of the fixes are all in one place.

**Pull request closed without merging**